src/Parsers/Kusto/KustoFunctions/KQLDateTimeFunctions.cpp

```diff
@@ -102,16 +112,18 @@ bool DatetimePart::convertImpl(String & out, IParser::Pos & pos)
     if (part.front() == '\"' || part.front() == '\'')
     {
         //period.remove
-        part.erase( 0, 1); // erase the first quote
-        part.erase( part.size() - 1); // erase the last quote
+        part.erase(0, 1); // erase the first quote
+        part.erase(part.size() - 1); // erase the last quote
     }
     String date;
     if (pos->type == TokenType::Comma)
     {
-         ++pos;
-         date = getConvertedArgument(fn_name, pos);
+        ++pos;
+        date = getConvertedArgument(fn_name, pos);
     }
     String format;
+    String head = "";
+    String trail = "";
```

Comment:
You could just use their default constructors and it'd amount to the same thing:
```suggestion
    String head;
    String trail;
```